stories/control/timeaxis.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, LineSeries, Plot } from '../../src';
+import { DEFAULT_PLOT_CONFIG } from '../utils';
+
+export default {
+  title: 'API/TimeAxis',
+  args: {
+    label: 'Label',
+    hiddenSecondaryTicks: false,
+    hidden: false,
+    hiddenTicks: false,
+  },
+} as Meta;
+
+interface AxisControlProps {
+  label: string;
+  paddingStart: number;
+  paddingEnd: number;
+}
+
+const dataX = [
+  { x: new Date(2002, 11, 24, 10, 33), y: 10 },
+  { x: new Date(2002, 11, 24, 11, 33), y: 20 },
+  { x: new Date(2002, 11, 24, 12, 33), y: 40 },
+  { x: new Date(2002, 11, 24, 13, 33), y: 7 },
+  { x: new Date(2002, 11, 24, 14, 33), y: 15 },
+];
+const dataY = [
+  { x: 3, y: new Date(2002, 11, 24, 10, 33) },
+  { x: 6, y: new Date(2002, 11, 24, 11, 33) },
+  { x: 12, y: new Date(2002, 11, 24, 12, 33) },
+  { x: 14, y: new Date(2002, 11, 24, 13, 33) },
+  { x: 15, y: new Date(2002, 11, 24, 14, 33) },
```

Comment:
If it's supposed to work with numbers (epochs) as well, can you use numbers on one fothe axis?

Reply:
yes you can use numbers but it must be in milliseconds
check nasdaq example